lib/modules/manager/mint/__fixtures__/Mintfile

```diff
@@ -0,0 +1,4 @@
+SwiftGen/SwiftGen@6.6.1
```

Comment:
why this still isn't inlined?

Reply:
I'm so sorry! Thanks for all your reviews! I implemented with this commit https://github.com/renovatebot/renovate/pull/17351/commits/a3c052fbb9a3923022b97de7c93e31fc8ae5b53c